aws/resource_aws_sfn_activity.go

```diff
@@ -34,6 +35,7 @@ func resourceAwsSfnActivity() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"tags": tagsSchema(),
```

Comment:
This new attribute is missing drift detection in the `Read` function. It can be added similar to the state machine read logic below (with its update).

Reply:
Hey @sunilkumarmohanty -- not sure if you are done making changes or didn't push something up, but you marked this as resolved and I don't see any changes to `resourceAwsSfnActivityRead()`, e.g. adding

```go
  tagsResp, err := conn.ListTagsForResource(
    &sfn.ListTagsForResourceInput{
      ResourceArn: aws.String(d.Id()),
    },
  )

  if err != nil {
    return fmt.Errorf("error listing SFN Activity (%s) tags: %s", d.Id(), err)
  }

  if err := d.Set("tags", tagsToMapSfn(tagsResp.Tags)); err != nil {
    return fmt.Errorf("error setting tags: %s", err)
  }
```

Reply:
I am sorry, I forgot to make these changes. It is done now.